extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/cors/CORSFilter.java

```diff
@@ -36,6 +39,44 @@ public static boolean isConfiguredWithWildcard(Optional<List<String>> optionalLi
         return list.isEmpty() || (list.size() == 1 && "*".equals(list.get(0)));
     }
 
+    /**
+     * Parse the provided allowed origins for any regexes
+     * 
+     * @param allowedOrigins
+     * @return a list of compiled regular expressions. If none configured, and empty list is returned
+     */
+    public static List<Pattern> parseAllowedOriginsRegex(Optional<List<String>> allowedOrigins) {
+        if (allowedOrigins == null || !allowedOrigins.isPresent()) {
+            return Collections.emptyList();
+        }
+
+        // extract configured origins and find any Regular Expressions
+        List<Pattern> allowOriginsRegex = new ArrayList<>();
+        allowedOrigins.get().stream()
```

Comment:
@ekasumov Thanks, can you replace with a for loop ? This filter is initialized from the recorder - so it is not recommended to use the lambda in such cases AFAIK.
Can also please add a small note to the docs explaining how to set the reg expression ?
thanks

Reply:
@radcortez Thanks, I forgot there is no dedicated property yet exists for Regex origins so for now it won't work.

Reply:
@sberyozkin should all lambdas in this class be replaced, or just the one in the parseAllowedOriginsRegex() method?

Reply:
@ekasumov IMHO this is fine for now just to keep this PR not affecting other code.
CORS filter is documented [here](https://github.com/quarkusio/quarkus/blob/main/docs/src/main/asciidoc/http-reference.adoc). I propose to update `quarkus.http.cors.origins`'s `Description` that `the values starting and ending with `/` are treated as regular expressions`, and copy/paste the example below the table showing reg-ex origins
thanks

Reply:
@sberyozkin done!